crates/humanode-runtime/src/lib.rs

```diff
@@ -563,11 +563,21 @@ impl pallet_bootnodes::Config for Runtime {
     type MaxBootnodes = ConstU32<16>;
 }
 
+impl pallet_fixed_validators_set::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type ValidatorId = AccountId;
+    type MaxValidators = ConstU32<MAX_AUTHENTICATIONS>; /* same amount in theory */
```

Comment:
Should we add `MaxBootnodes` number as well?

Reply:
Yeah, this fixed set doesn't replace the bootnodes - it is an addition rather